paasta_tools/cli/utils.py

```diff
@@ -877,6 +877,32 @@ def get_namespace_for_secret(
     return secret_to_k8s_namespace
 
 
+def select_k8s_secret_namespace(namespaces: Set[str]) -> Optional[str]:
+    namespaces_count = len(namespaces)
+
+    if not namespaces_count:
+        return None
+
+    if namespaces_count == 1:
+        return namespaces.pop()
+
+    prioritized_k8s_namespaces = [
+        "paasta",
+        "tron",
+        "paasta-flinks",
+        "paasta-cassandraclusters",
+        "paasta-kafkaclusters",
+        "paasta-nrtsearchservices",
```

Comment:
It's not great to have these hardcoded here, it'd be better to load from some configuration.

Also know that services are eventually moving to individual namespaces prefixed with `paastasvc-`, so it'll need changing (PAASTA-17769).